vlib/v/checker/tests/match_type_node_with_non_sum_type_err.out

```diff
@@ -0,0 +1,7 @@
+vlib/v/checker/tests/match_type_node_with_non_sum_type_err.vv:5:3: error: `match` by type can only be used for sum-types, `b` is of type `i64`
```

Comment:
it's not only for sum types, so this part has to be removed

Reply:
Fixed